.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish cloud-sdk 📦 to PyPI and TestPyPI
+on: 
+  push:
+    branches:
```

Comment:
We publish to pypi everytime we merge in dev or master ?

Reply:
I feel like we'll need regular changes to the library for our other services and we'll need to publish the new package for the changes to take effect. It can be cumbersome to have to merge everything to dev then create a PR from dev to master for minimal changes (e.g; my one line changes in the last PRs).

As soon as the SDK is "stable" we'll publish only on the master branch to PYPI. 


Thinking out loud, maybe we could publish to Test PyPI repository on dev and PYPI on master and add an env variable to let the end-to-end testing framework chose the package (and repository) it wants to pull. I think that could be pretty cool later on but that's a bit overkill now since we are the only user of the SDK for now.

Reply:
I have never used the testpypi... Your plan sounds reasonable to me. Publish from dev for now, and from master later on.
In that case, should we disable publishing from master for now ?